### ISSUE TITLE
bug in find_symmetry utility

### ISSUE THREAD

**Issue by fkraushofer** [viperleed/viperleed#273]
I just tried using the find_symmetry utility (input: `viperleed poscar find_symmetry < POSCAR`) and get the following error:

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\fkrau\anaconda3\envs\viperleed\Scripts\viperleed.exe\__main__.py", line 7, in <module>
  File "C:\Users\fkrau\anaconda3\envs\viperleed\Lib\site-packages\viperleed\cli_base.py", line 423, in run_as_script
    exit_code = cli()
                ^^^^^
  File "C:\Users\fkrau\anaconda3\envs\viperleed\Lib\site-packages\viperleed\cli_base.py", line 256, in __call__
    return command(args)
           ^^^^^^^^^^^^^
  File "C:\Users\fkrau\anaconda3\envs\viperleed\Lib\site-packages\viperleed\utilities\poscar\base.py", line 45, in __call__
    slab = self.read_poscar(parsed_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\fkrau\anaconda3\envs\viperleed\Lib\site-packages\viperleed\utilities\poscar\base.py", line 194, in read_poscar
    raise AttributeError('--infile argument is missing. If you have '
AttributeError: --infile argument is missing. If you have overridden add_parser_arguments without also overriding read_poscar, make sure to call add_infile_argument in your overridden add_parser_arguments
```

This is with the current version of the master branch (i.e. recently installed with `pip install viperleed`). Environment is Windows, and Python 3.12.7 (using Anaconda).

I briefly tested `delete_above` to see whether this is a more general problem with the POSCAR utilities, but that one works fine.

Probably something for @amimre?

**Comment by amimre** [comment]
Hm, I see, I'll look into it.

**Comment by michele-riva** [comment]
Actually, easier to write it here. In https://github.com/viperleed/viperleed/blob/859c3bb567db99f8efb0d247e31b7a1e9c2fe545/src/viperleed/utilities/poscar/base.py#L246-L270

there's a missing `super().add_parser_arguments(parser)` call (the placement at the beginning or at the end only changes the order of the CLI arguments).

**Comment by michele-riva** [comment]
Fixed by #274 in 0.12.2
